docs/outlook/one-outlook.md

```diff
@@ -0,0 +1,81 @@
+ms.topic: article
+ms.date: 02/07/2023
+ms.localizationpriority: medium
+---
+
+# Develop Outlook add-ins for the new Outlook on Windows (preview)
+
+The new Outlook on Windows desktop client unifies the Windows and web codebases to create a more consistent Outlook experience for users and administrators. Its modern and simplified interface has added capabilities and aims to improve productivity, organization, and collaboration for users. More importantly, the new Outlook on Windows supports Outlook web add-ins, so that you can continue to extend Outlook's functionality.
+
+## Impact on VSTO and COM add-ins
+
+Because the new Outlook on Windows is based on the Outlook on the web codebase, VSTO and COM add-ins aren't supported. To ensure your add-in continues to work in the new Outlook on Windows, you must migrate your VSTO or COM add-in to an Outlook web add-in. Migrating to an Outlook web add-in not only enables compatibility with the new Outlook on Windows; it also makes your solution available to users on other platforms, such as Outlook on Mac, on mobile, or on the web.
+
+To help get you started on the migration process, review the following guidance.
+
+- The differences in features and scenarios supported by VSTO and COM add-ins and Outlook web add-ins are being addressed. To determine whether your add-in scenario is fully supported in an Outlook web add-in, see [Supported scenarios in Outlook web add-ins](#supported-scenarios-in-outlook-web-add-ins).
+- For guidance on how to transition your VSTO add-in to an Outlook web add-in, see [VSTO add-in developer's guide](../overview/learning-path-transition.md) and [Tutorial: Share code between both a VSTO Add-in and an Office Add-in with a shared code library](../tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md).
+- If you're new to Outlook web add-ins, try out the [Outlook quick start](../quickstarts/outlook-quickstart.md) to build your first add-in.
+
+> [!NOTE]
+>
+> VSTO and COM add-ins are still supported in classic Outlook on Windows.
+
+### Supported scenarios in Outlook web add-ins
+
+The development of the Outlook JavaScript API used by Outlook web add-ins is focused on closing the gap on scenarios that are only supported by VSTO and COM add-in solutions. This way, users who transition to the Outlook web add-in can continue to have a seamless experience.
+
+The following table identifies key Outlook scenarios and their support status in a web add-in. This table will be updated as additional scenarios are supported. Periodically check this section as you plan to migrate your VSTO or COM add-in.
+
+|Scenario|Description|Support status in Outlook web add-ins|
+|-----|-----|-----|
+|Spam email reporting and education|Enable users to report unsolicited and potentially unsafe messages and learn how to identify these messages.|Supported. Improvements are in development to further enhance the user experience.|
```

Comment:
Consider alphabetizing the scenarios

Reply:
I'll take this into consideration for a future iteration as the current order is based on most popular scenarios.